### PR TITLE
Configure Unit-Test to run with CI

### PR DESCRIPTION
**Pull request by asifdxtreme**
Signed-off-by: asifdxtreme <mohammad.asif.siddiqui1@huawei.com>

### CODE REVIEW

File: util/fileutil/fileutil.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func GetConfDir() string {
 		if err != nil {
 			log.Panic(err)
 		}
-		log.Println("Conf dir is "+configDir)
+		log.Println("Conf dir is " + configDir)
```
Comment (tianxiaoliang) [inline util/fileutil/fileutil.go]: not go fmt

**Comment by asifdxtreme** [comment]
For all the commented code we have raised an issue https://github.com/ServiceComb/go-chassis/issues/5 to fix this later

**Comment by tianxiaoliang** [comment]
ok, but there is conflicts

**Comment by asifdxtreme** [comment]
@tianxiaoliang successfully resolved all the conflicts, please review and merge

**Pull request merged**